README.md

```diff
@@ -786,6 +786,13 @@ seed_jobs:
       pollScm: '* * * * *'
       # period trigger
       periodic: '* * * * *'
+      # Artifactory trigger
+      # To use this trigger, Artifactory should be created also by using 'Artifactory' section.
+      artifactory:
+        serverId: artName #Name of Artifactory, same as created Artifactory
+        artifactoryUrl: https://mycompany.com/artifactory #Url of Artifactory, same as created Artifactory
```

Comment:
Can't you get the url from the artifactory serverId by accessing its object? This configuration will result in specifying the artifactory url twice

Reply:
This is because the way the constructor is created in Artifactory plugin.

Reply:
If you look at https://github.com/jenkinsci/artifactory-plugin/blob/master/src/main/java/org/jfrog/hudson/trigger/ArtifactoryTrigger.java#L49
You will see that it only uses the serverId (i.e. `details.getArtifactoryName()`)

Reply:
You are right, removed...